services/api/src/lib/algoexplorer-adapter.ts

```diff
@@ -0,0 +1,54 @@
+import got from 'got'
+
+import { Configuration } from '@/configuration'
+import { logger } from '@/utils/logger'
+
+/**
+ * AlgoExplorer APIv2 Docs:
+ * https://testnet.algoexplorer.io/api-dev/v2
+ */
+
+interface AlgoExplorerAsset {
+  amount: number
+  'asset-id': number
+  creator: string
+  'is-frozen': boolean
+}
+
+interface AlgoExplorerAccount {
+  address: string
+  amount: number
```

Comment:
Good question. We don't store or use anything related to this value in the application. I (partially) typed interface this based on the response we get from AlgoExplorer. To avoid confusion, maybe it's best to only type the parts we _do_ use. I'll make that change unless there are any objections.